### DISCUSSION TITLE
Comments in a TXT template?

### DISCUSSION THREAD

**Discussion by cziesman** [thymeleaf/thymeleaf#973] [Q&A]
Can comments be included in a TXT template? I tried both of these (which work with HTML) without success.

```
<!--/*-->
    Licensed to the Apache Software Foundation (ASF) under one or more
    contributor license agreements.  See the NOTICE file distributed with
    this work for additional information regarding copyright ownership.
    The ASF licenses this file to You under the Apache License, Version 2.0
    (the "License"); you may not use this file except in compliance with
    the License.  You may obtain a copy of the License at

         http://www.apache.org/licenses/LICENSE-2.0

    Unless required by applicable law or agreed to in writing, software
    distributed under the License is distributed on an "AS IS" BASIS,
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
<!--*/-->
<!--/* This code will be removed at thymeleaf parsing time! */-->
```

**Comment by cziesman** [comment]
Further experimentation shows that the file suffix is significant. If I name my TXT files with a `.txt` suffix, then the comments are not removed, whether I use a template mode of `TEXT` or `HTML`. If I name my TXT files with a `.html` suffix, then the comments are removed for both template modes. So, it seems that template mode is basically ignored, and the file suffix actually determines how the files are processed. Is that correct?

**Reply by PedroFernandesGenesis** [reply]
Any answers? Trying to add a comment on my .txt file without success

**Comment by landsman** [comment]
Why is it necessary to spam every template file license? You should only need one LICENCE file attached to the project, in root. This is a sick practice.

**Comment by PedroFernandesGenesis** [comment]
In my case I needed some comments in the middle of the document for organization purpose. I was able to do it using  the following 
`[# th:comment=" ----- Block 1 ----- "][/]`
This is not appearing on results
